src/views/NewLanding/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react';
+
+import {
+  Button,
+  Grid,
+  Column
+} from '@carbon/react';
+import { Login } from '@carbon/icons-react';
+import { KeycloakLoginOptions } from 'keycloak-js';
+import { useNavigate } from 'react-router-dom';
+import { useAuth } from '../../contexts/AuthContext';
+import BCGovLogo from '../../components/BCGovLogo';
+import Seeding from '../../assets/img/seeding.png';
+import LoginProviders from '../../types/LoginProviders';
+import './styles.css';
+import getUrlQueryParam from '../../utils/UrlUtils';
+
+const NewLanding = () => {
+  const { startKeycloak, login, signed } = useAuth();
+  const navigate = useNavigate();
+  const homePage = '/home';
+
+  const handleLogin = (provider: LoginProviders) => {
+    if (signed) {
+      navigate(getUrlQueryParam(window.location, 'page') || homePage);
+      return;
+    }
+
+    const idpHint = provider;
+    const loginOptions: KeycloakLoginOptions = {
+      redirectUri: `${window.location.origin}${homePage}`,
+      idpHint
+    };
+
+    login(loginOptions);
+  };
+
+  useEffect(() => {
+    if (signed) {
+      navigate(getUrlQueryParam(window.location, 'page') || homePage);
+    } else {
+      startKeycloak();
+    }
+  }, [signed]);
+
+  return (
+    <Grid fullWidth>
+      {/* First - Column */}
+      <Column lg={8}>
+        {/* Logo */}
+        <BCGovLogo />
+
+        {/* Welcome - Title and Subtitle */}
+        <h1 data-testid="landing-title" className="landing-title">Welcome to SPAR</h1>
+        <p data-testid="landing-subtitle" className="landing-subtitle">
```

Comment:
Since it's a subtitle, shouldn't this be a heading element (`h2`, in this case, since it would be the second level)?

A screen reader (I don't know exactly how it's called, I'll call it reader by now) would interpret this as common text — a simple paragraph, which is not what it is.

Reply:
Great point. Fixed.